exams/api.py

```diff
@@ -120,6 +120,7 @@ def authorize_for_exam_run(user, course_run, exam_run):
         user=mmtrack.user,
         course=course_run.course,
         exam_run=exam_run,
+        status=ExamAuthorization.STATUS_SUCCESS,
```

Comment:
Should this be in a defaults dict: `defaults={"status": ExamAuthorization.STATUS_SUCCESS}`? I think you would have multiple conflicting `ExamAuthorization` otherwise